prepline_general/api/general.py

```diff
@@ -90,28 +91,31 @@ def get_pdf_splits(pdf, split_size=1):
     return split_pdfs
 
 
-def partition_file_via_api(file_tuple, **kwargs):
+def partition_file_via_api(file_tuple, request, filename, content_type, **request_kwargs):
```

Comment:
maybe `request_kwargs` could be renamed to `form_parameter_kwargs`? this only pertains to form parameters, right?

The docstring below should be updated, might as will include comments for the kwargs there.

Reply:
Good catch - updated the docstrings. I renamed this to `partition_kwargs` to clarify that these are parameters that will be passed on, or sent to local partition. This allows me to exclude things like `coordinates`, which will be forwarded, but not passed to local partition.